packages/api/src/feature_flag/controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Router } from "express";
+import type { Request, Response, NextFunction } from "express";
+
+import { featureService } from "./service";
+import { extractUserIsAdminFromAuthToken } from "../middleware";
+import { validateIsAdminFromAuthentication } from "../validators/shared";
+
+export const featureController = Router();
+
+featureController.get(
+  "/",
+  extractUserIsAdminFromAuthToken,
+  async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+      validateIsAdminFromAuthentication(req.body);
```

Comment:
Now that we have a validator in this controller, we should add back the check for validation errors (and write a test that we return the correct status code in case of a validation error).